### ISSUE TITLE
Problem with relogin

### ISSUE THREAD

**Issue by Gnacik** [OpenPoGo/OpenPoGoBot#168]
Bot have problems with re-login, so it crash occasionally after some time.

```
[API] Token has expired, attempting to log back in...
Traceback (most recent call last):
  File "pokecli.py", line 316, in <module>
    main()
  File "pokecli.py", line 309, in main
    bot.run()
  File "/home/developer/OpenPoGoBot/pokemongo_bot/__init__.py", line 92, in run
    self.navigator.navigate(map_cells)
  File "/home/developer/OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py", line 37, in navigate
    self.stepper.walk_to(*position)
  File "/home/developer/OpenPoGoBot/pokemongo_bot/stepper.py", line 60, in walk_to
    self._walk_to(to_lat, to_lng, alt)
  File "/home/developer/OpenPoGoBot/pokemongo_bot/stepper.py", line 78, in _walk_to
    self._jump_to(c_lat, c_long, to_alt)
  File "/home/developer/OpenPoGoBot/pokemongo_bot/stepper.py", line 111, in _jump_to
    map_cells = self.bot.mapper.get_cells(self.current_lat, self.current_lng)
  File "/home/developer/OpenPoGoBot/pokemongo_bot/mapper.py", line 34, in get_cells
    response_dict = self.api_wrapper.call()
  File "/home/developer/OpenPoGoBot/api/__init__.py", line 73, in call
    if self.login() is False:
  File "/home/developer/OpenPoGoBot/api/__init__.py", line 29, in login
    return self._api.login(provider, username, password, app_simulation=True)
  File "/home/developer/OpenPoGoBot/src/pgoapi/pgoapi/pgoapi.py", line 126, in login
    response = request.call()
  File "/home/developer/OpenPoGoBot/src/pgoapi/pgoapi/pgoapi.py", line 182, in call
    response = request.request(self._api_endpoint, self._req_method_list, self.get_position())
  File "/home/developer/OpenPoGoBot/src/pgoapi/pgoapi/rpc_api.py", line 126, in request
    raise NotLoggedInException()
pgoapi.exceptions.NotLoggedInException
```

**Comment by Gnacik** [comment]
Secong problem with relogin, i will not open new issue, just put it here.
Today happens often:

```
[API] Token has expired, attempting to log back in...
2016-08-02 12:36:48,084 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-02 12:37:04,257 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-02 12:37:20,543 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-02 12:37:36,721 [    pgoapi] [ERROR] Login failed - unexpected server response!
```

after Ctrl-c and start again it connect without problems, and working.

**Comment by lazyfai** [comment]
Got similar issue, usually it take around 5-10 retries and it will be back online, but this will also trigger a logout on all my android devices... I am trying to dump the "server reponse" out to see what happened.

**Comment by lazyfai** [comment]
Created a new account for testing, while this login failed message happened, the reponse from server is like this: (while the python code is looking for an URL for RPC, which is missed)

> {
>   "responses": {
>     "GET_HATCHED_EGGS": {
>       "success": true
>     },
>     "GET_INVENTORY": {
>       "inventory_delta": {
>         "inventory_items": [
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 16,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470150629630
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 221,
>                 "move_2": 26,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 74,
>                 "creation_time_ms": 1470151232167,
>                 "height_m": 0.3470310866832733,
>                 "stamina_max": 15,
>                 "weight_kg": 15.130386352539062,
>                 "individual_defense": 1,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 15,
>                 "individual_stamina": 14,
>                 "individual_attack": 4,
>                 "cp": 32,
>                 "id": 9114419363596190000
>               }
>             },
>             "modified_timestamp_ms": 1470151232180
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 219,
>                 "move_2": 50,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 41,
>                 "creation_time_ms": 1470151194032,
>                 "height_m": 0.685101330280304,
>                 "stamina_max": 15,
>                 "weight_kg": 4.03708028793335,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 15,
>                 "individual_stamina": 13,
>                 "individual_attack": 5,
>                 "cp": 23,
>                 "id": 5572818897764671000
>               }
>             },
>             "modified_timestamp_ms": 1470151194044
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "is_egg": true,
>                 "egg_incubator_id": "EggIncubatorProto-3810835877979407990",
>                 "egg_km_walked_target": 5,
>                 "creation_time_ms": 1470151734068,
>                 "captured_cell_id": 3748115441607770000,
>                 "id": 4363442215236040000
>               }
>             },
>             "modified_timestamp_ms": 1470151753757
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 237,
>                 "move_2": 131,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 60,
>                 "creation_time_ms": 1470153138384,
>                 "height_m": 0.6487046480178833,
>                 "stamina_max": 13,
>                 "weight_kg": 12.277740478515625,
>                 "individual_defense": 4,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 13,
>                 "individual_attack": 13,
>                 "cp": 30,
>                 "id": 18201426845759162000
>               }
>             },
>             "modified_timestamp_ms": 1470153138399
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 32
>               }
>             },
>             "modified_timestamp_ms": 1470152703459
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 2,
>                 "times_encountered": 2,
>                 "pokemon_id": 35
>               }
>             },
>             "modified_timestamp_ms": 1470151205431
>           },
>           {
>             "inventory_item_data": {
>               "item": {
>                 "item_id": 901,
>                 "count": 1,
>                 "unseen": true
>               }
>             },
>             "modified_timestamp_ms": 1470150376742
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "creation_time_ms": 1470152765931,
>                 "captured_cell_id": 3748115442276761600,
>                 "is_egg": true,
>                 "id": 10364541863967820000,
>                 "egg_km_walked_target": 2
>               }
>             },
>             "modified_timestamp_ms": 1470152765931
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 32,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470152703459
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 2,
>                 "times_encountered": 2,
>                 "pokemon_id": 41
>               }
>             },
>             "modified_timestamp_ms": 1470152497036
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 35,
>                 "candy": 7
>               }
>             },
>             "modified_timestamp_ms": 1470151675856
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "creation_time_ms": 1470152404068,
>                 "captured_cell_id": 3748115442102698000,
>                 "is_egg": true,
>                 "id": 9515815110908768000,
>                 "egg_km_walked_target": 5
>               }
>             },
>             "modified_timestamp_ms": 1470152404068
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 221,
>                 "move_2": 133,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 10,
>                 "creation_time_ms": 1470152790002,
>                 "height_m": 0.30801719427108765,
>                 "stamina_max": 10,
>                 "weight_kg": 2.514163017272949,
>                 "individual_defense": 7,
>                 "cp_multiplier": 0.09399999678134918,
>                 "stamina": 10,
>                 "individual_stamina": 2,
>                 "cp": 10,
>                 "id": 18190497356236315000
>               }
>             },
>             "modified_timestamp_ms": 1470152790040
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 41,
>                 "candy": 6
>               }
>             },
>             "modified_timestamp_ms": 1470152497036
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "creation_time_ms": 1470151882232,
>                 "captured_cell_id": 3748115442425659400,
>                 "is_egg": true,
>                 "id": 9797341805836804000,
>                 "egg_km_walked_target": 5
>               }
>             },
>             "modified_timestamp_ms": 1470151882232
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 116
>               }
>             },
>             "modified_timestamp_ms": 1470152645689
>           },
>           {
>             "inventory_item_data": {
>               "player_stats": {
>                 "pokemons_captured": 12,
>                 "km_walked": 1.7900021076202393,
>                 "level": 4,
>                 "experience": 8050,
>                 "pokemons_encountered": 12,
>                 "pokeballs_thrown": 15,
>                 "unique_pokedex_entries": 10,
>                 "prev_level_xp": 3000,
>                 "next_level_xp": 10000,
>                 "pokemon_caught_by_type": "AAEAAwMBAQEAAAACAAABAAAAAg==",
>                 "poke_stop_visits": 15
>               }
>             },
>             "modified_timestamp_ms": 1470153267000
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 211,
>                 "move_2": 126,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 32,
>                 "creation_time_ms": 1470152703444,
>                 "height_m": 0.5433390140533447,
>                 "stamina_max": 10,
>                 "weight_kg": 9.45565128326416,
>                 "individual_defense": 15,
>                 "cp_multiplier": 0.09399999678134918,
>                 "stamina": 10,
>                 "individual_stamina": 3,
>                 "individual_attack": 7,
>                 "cp": 10,
>                 "id": 13238587595721966000
>               }
>             },
>             "modified_timestamp_ms": 1470152703459
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 221,
>                 "move_2": 105,
>                 "captured_cell_id": 3748115449004425000,
>                 "pokemon_id": 7,
>                 "creation_time_ms": 1470150546528,
>                 "height_m": 0.6300575733184814,
>                 "stamina_max": 10,
>                 "weight_kg": 15.150253295898438,
>                 "individual_defense": 10,
>                 "cp_multiplier": 0.09399999678134918,
>                 "stamina": 10,
>                 "individual_stamina": 10,
>                 "individual_attack": 10,
>                 "cp": 13,
>                 "id": 17867715106035692000
>               }
>             },
>             "modified_timestamp_ms": 1470150546528
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 116,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470152645689
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 60
>               }
>             },
>             "modified_timestamp_ms": 1470153138399
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 63
>               }
>             },
>             "modified_timestamp_ms": 1470152299109
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 222,
>                 "move_2": 84,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 35,
>                 "creation_time_ms": 1470151058671,
>                 "height_m": 0.6562016606330872,
>                 "stamina_max": 13,
>                 "weight_kg": 9.648897171020508,
>                 "individual_defense": 14,
>                 "cp_multiplier": 0.09399999678134918,
>                 "stamina": 13,
>                 "individual_stamina": 4,
>                 "individual_attack": 5,
>                 "cp": 15,
>                 "id": 14690910336494217000
>               }
>             },
>             "modified_timestamp_ms": 1470151058685
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 219,
>                 "move_2": 90,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 41,
>                 "creation_time_ms": 1470152497024,
>                 "height_m": 0.9220840930938721,
>                 "stamina_max": 14,
>                 "weight_kg": 9.893206596374512,
>                 "individual_defense": 5,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 14,
>                 "individual_stamina": 5,
>                 "individual_attack": 10,
>                 "cp": 24,
>                 "id": 13439812769851013000
>               }
>             },
>             "modified_timestamp_ms": 1470152497036
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 234,
>                 "move_2": 99,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 63,
>                 "creation_time_ms": 1470152299097,
>                 "height_m": 0.9018675684928894,
>                 "stamina_max": 10,
>                 "weight_kg": 20.73248291015625,
>                 "individual_defense": 15,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 10,
>                 "individual_attack": 1,
>                 "cp": 20,
>                 "id": 17933453164472230000
>               }
>             },
>             "modified_timestamp_ms": 1470152299109
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 60,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470153138399
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 7
>               }
>             },
>             "modified_timestamp_ms": 1470150546528
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 63,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470152299109
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 230,
>                 "move_2": 36,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 116,
>                 "creation_time_ms": 1470152645675,
>                 "height_m": 0.410139262676239,
>                 "stamina_max": 11,
>                 "weight_kg": 8.463544845581055,
>                 "individual_defense": 4,
>                 "cp_multiplier": 0.16639786958694458,
>                 "stamina": 11,
>                 "individual_stamina": 11,
>                 "individual_attack": 15,
>                 "cp": 32,
>                 "id": 297256762301513300
>               }
>             },
>             "modified_timestamp_ms": 1470152645689
>           },
>           {
>             "inventory_item_data": {
>               "item": {
>                 "item_id": 1,
>                 "count": 76
>               }
>             },
>             "modified_timestamp_ms": 1470153267000
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "move_1": 219,
>                 "move_2": 45,
>                 "captured_cell_id": 3748115292320956400,
>                 "pokeball": 1,
>                 "pokemon_id": 16,
>                 "creation_time_ms": 1470150629614,
>                 "height_m": 0.3258682191371918,
>                 "stamina_max": 10,
>                 "weight_kg": 2.2906951904296875,
>                 "individual_defense": 10,
>                 "cp_multiplier": 0.09399999678134918,
>                 "stamina": 10,
>                 "individual_stamina": 3,
>                 "individual_attack": 4,
>                 "cp": 10,
>                 "id": 12089219957497600000
>               }
>             },
>             "modified_timestamp_ms": 1470150629630
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 74
>               }
>             },
>             "modified_timestamp_ms": 1470151232180
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 10
>               }
>             },
>             "modified_timestamp_ms": 1470152790040
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 7,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470150546528
>           },
>           {
>             "inventory_item_data": {
>               "pokedex_entry": {
>                 "times_captured": 1,
>                 "times_encountered": 1,
>                 "pokemon_id": 16
>               }
>             },
>             "modified_timestamp_ms": 1470150629630
>           },
>           {
>             "inventory_item_data": {
>               "egg_incubators": {
>                 "egg_incubator": [
>                   {
>                     "pokemon_id": 4363442215236040000,
>                     "start_km_walked": 0.41122522950172424,
>                     "target_km_walked": 5.411225229501724,
>                     "incubator_type": 1,
>                     "item_id": 901,
>                     "id": "EggIncubatorProto-3810835877979407990"
>                   }
>                 ]
>               }
>             },
>             "modified_timestamp_ms": 1470151753757
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 74,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470151232180
>           },
>           {
>             "inventory_item_data": {
>               "pokemon_data": {
>                 "creation_time_ms": 1470152539703,
>                 "captured_cell_id": 3748115441955897300,
>                 "is_egg": true,
>                 "id": 8742598072869640000,
>                 "egg_km_walked_target": 2
>               }
>             },
>             "modified_timestamp_ms": 1470152539703
>           },
>           {
>             "inventory_item_data": {
>               "candy": {
>                 "family_id": 10,
>                 "candy": 3
>               }
>             },
>             "modified_timestamp_ms": 1470152790040
>           },
>           {
>             "inventory_item_data": {
>               "item": {
>                 "item_id": 401,
>                 "count": 2,
>                 "unseen": true
>               }
>             },
>             "modified_timestamp_ms": 1470150376742
>           }
>         ],
>         "new_timestamp_ms": 1470153418201
>       },
>       "success": true
>     },
>     "DOWNLOAD_SETTINGS": {
>       "hash": "80f4f3719421640d132515f1c293fe0455ae8d53",
>       "settings": {
>         "fort_settings": {
>           "far_interaction_range_meters": 1000,
>           "max_total_deployed_pokemon": 10,
>           "max_player_deployed_pokemon": 1,
>           "deploy_stamina_multiplier": 2,
>           "interaction_range_meters": 40
>         },
>         "inventory_settings": {
>           "base_bag_items": 350,
>           "max_pokemon": 1000,
>           "base_eggs": 9,
>           "max_bag_items": 1000,
>           "base_pokemon": 250
>         },
>         "map_settings": {
>           "get_map_objects_max_refresh_seconds": 30,
>           "get_map_objects_min_distance_meters": 10,
>           "encounter_range_meters": 50,
>           "poke_nav_range_meters": 201,
>           "pokemon_visible_range": 70,
>           "get_map_objects_min_refresh_seconds": 5,
>           "google_maps_api_key": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
>         },
>         "minimum_client_version": "0.29.0"
>       }
>     },
>     "GET_PLAYER": {
>       "player_data": {
>         "username": "LazyFai",
>         "equipped_badge": {},
>         "currencies": [
>           {
>             "name": "POKECOIN"
>           },
>           {
>             "amount": 1200,
>             "name": "STARDUST"
>           }
>         ],
>         "tutorial_state": [
>           0,
>           1,
>           3,
>           4,
>           7
>         ],
>         "contact_settings": {},
>         "max_item_storage": 350,
>         "creation_timestamp_ms": 1470150376742,
>         "daily_bonus": {},
>         "avatar": {
>           "eyes": 2,
>           "shoes": 1,
>           "shirt": 1,
>           "backpack": 2,
>           "hair": 2,
>           "skin": 1,
>           "hat": 2,
>           "pants": 1
>         },
>         "max_pokemon_storage": 250
>       },
>       "success": true
>     },
>     "CHECK_AWARDED_BADGES": {
>       "success": true
>     }
>   },
>   "status_code": 1,
>   "auth_ticket": {
>     "expire_timestamp_ms": 1470155388086,
>     "start": "B2sS4dXSYfP+/Ene8vlezpAddCP7sSPP36gKatIo3b3SW8DcVQEAjiVVCy/tj5EtQNVfo+KJLXgJnWAjXmb6lw==",
>     "end": "lnrYV+D8kjEyj38Q0Oi2dg=="
>   },
>   "request_id": 874521875742250100
> }

**Comment by alexandre-leites** [comment]
I have the same issue. I can help solving if you guys give me directions how to debug.

**Comment by Dimitrie78** [comment]
[2016-08-03 21:55:04] [Navigation] Walking towards PokeStop "Neugotischer Turm" (73.78m away, eta 1.00 minutes)
[API] Token has expired, attempting to log back in...
2016-08-03 21:55:24,973 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:55:41,540 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:55:58,042 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:56:14,690 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:56:31,230 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:56:47,815 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:57:04,375 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:57:20,880 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:57:37,452 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
2016-08-03 21:57:54,175 [    pgoapi] [ERROR] Login failed - unexpected server response!
[API] Failed to login. Waiting 15 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
[API] API call failed. Retrying in 10 seconds...
Traceback (most recent call last):
  File "./pokecli.py", line 347, in <module>
    main()
  File "./pokecli.py", line 340, in main
    bot.run()
  File "/var/www/html/OpenPoGoBot/pokemongo_bot/**init**.py", line 103, in run
    self.mapper.get_cells_at_current_position()
  File "/var/www/html/OpenPoGoBot/pokemongo_bot/**init**.py", line 113, in work_on_cells
    for cell in map_cells:
TypeError: 'NoneType' object is not iterable

**Comment by Rlewisrlou666** [comment]
getting this error for the last few hours.

![snip20160803_8](https://cloud.githubusercontent.com/assets/20677921/17384778/397c51fe-59d6-11e6-8015-b0bdf3d51c23.png)

**Comment by wchill** [comment]
Fixed in #202.
